spec/models/claim_spec.rb

```diff
@@ -921,7 +922,7 @@
     end
 
     it "triggers validations on the eligibility appropriate to the context" do
```

Comment:
Again, should be in context under existing validations.

Reply:
@nomasprime As above, moved as we discussed